React-Calculator/src/components/WolframCalc.jsx

```diff
@@ -28,81 +28,92 @@ const getApiData = async (number1, operator, accumulator) => {
     throw new Error(`Error al calcular ${number1} ${operator} ${accumulator}`);
 };
 
+async function inputtedEquals({ number1, operator, accumulator, setAccumulator, setLogList }){
+    if(!operator){
+        return console.log("Nothing to operate")
```

Comment:
If this console.log wasn't created for testing purposes, I think I'd add a slightly more descriptive message and make it a `console.warn`, in order for the user to have a better idea of what to expect when it sees it, and so that they know they are probably doing something that they shouldn't do (that is, trying to submit an empty calculation)